README.rst

```diff
@@ -85,7 +85,7 @@ following command to install:
 
 .. code-block:: shell
 
-    circup install vl53l1x
+    circup install Adafruit_CircuitPython_VL53L1X
```

Comment:
```suggestion
    circup install adafruit_vl53l1x
```
This is the format normally used by circup (though both actually work, since circup translates pypi names).